### PR TITLE
Fix log forwarding options

### PR DESCRIPTION
**Pull request by devicezero**
## Why

The CLI options to add syslog forwarding are not updated yet

## What's changed

fixed typos and documentation for defaults

### CODE REVIEW

File: docs/src/increase-observability/logs/forward-logs.md [hunk 1]
```diff
@@ -110,20 +110,20 @@ You can forward your Platform.sh logs to any of those endpoints.
...
+| ---------------- | --------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `auth-token`     | `string`  |            | The token to authenticate with the given service.                                                                                                     |
+| `auth-mode`      | `string`  |            | The mode for authentication with the given service. Can be `prefix` or `structured_data`.                                                             |
 | `facility`       | `string`  | `1` (user) | A [syslog facility code](https://en.wikipedia.org/wiki/Syslog#Facility) to attach with each log to identify the source. Can be a number from 0 to 23. |
-| `message_format` | `string`  | `rfc5424`  | The standard to use for the message format. Can be `rfc5424` or `rfc3164`. |
-| `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`. |
```
Comment (CollierCZ) [inline docs/src/increase-observability/logs/forward-logs.md:125]: Are you sure about this? At least the help text indicates `tls` is still the default.

File: docs/src/increase-observability/logs/forward-logs.md [hunk 2]
```diff
@@ -110,20 +110,20 @@ You can forward your Platform.sh logs to any of those endpoints.
...
 
-| Property         | Type      | Default    | Description |
-| ---------------- | --------- | ---------- | ----------- |
-| `auth_token`     | `string`  |            | The token to authenticate with the given service. |
-| `auth_mode`      | `string`  |            | The mode for authentication with the given service. Can be `prefix` or `structured_data`. |
+| Property         | Type      | Default    | Description                                                                                                                                           |
+| ---------------- | --------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `auth-token`     | `string`  |            | The token to authenticate with the given service.                                                                                                     |
+| `auth-mode`      | `string`  |            | The mode for authentication with the given service. Can be `prefix` or `structured_data`.                                                             |
```
Comment (CollierCZ) [inline docs/src/increase-observability/logs/forward-logs.md:122]:
```suggestion
| `auth-mode`      | `string`  | `prefix`   | The mode for authentication with the given service. Can be `prefix` or `structured_data`. Defaults to `prefix`.                                        |
```

**Review by CollierCZ — CHANGES_REQUESTED** [review]

File: docs/src/increase-observability/logs/forward-logs.md (hunk 1 above)
Comment (CollierCZ) [reply docs/src/increase-observability/logs/forward-logs.md:125]: Seems to be: https://github.com/platformsh/legacy-cli/blob/main/src/Command/Integration/IntegrationCommandBase.php#L511

Comment (devicezero) [reply docs/src/increase-observability/logs/forward-logs.md:125]: you're right, I was too quick

File: docs/src/increase-observability/logs/forward-logs.md [hunk 3]
```diff
@@ -110,20 +110,20 @@ You can forward your Platform.sh logs to any of those endpoints.
...
 
-| Property         | Type      | Default    | Description |
-| ---------------- | --------- | ---------- | ----------- |
-| `auth_token`     | `string`  |            | The token to authenticate with the given service. |
-| `auth_mode`      | `string`  |            | The mode for authentication with the given service. Can be `prefix` or `structured_data`. |
+| Property         | Type      | Default    | Description                                                                                                                                           |
+| ---------------- | --------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `auth-token`     | `string`  |            | The token to authenticate with the given service.                                                                                                     |
+| `auth-mode`      | `string`  | `prefix`   | The mode for authentication with the given service. Can be `prefix` or `structured_data`. Defaults to `prefix`.                                        |
```
Comment (CollierCZ) [inline docs/src/increase-observability/logs/forward-logs.md:122]:
```suggestion
| `auth-mode`      | `string`  | `prefix`   | The mode for authentication with the given service. Can be `prefix` or `structured_data`. Defaults to `prefix`.                                       |
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 4]
```diff
@@ -110,20 +110,20 @@ You can forward your Platform.sh logs to any of those endpoints.
...
+| `auth-token`     | `string`  |            | The token to authenticate with the given service.                                                                                                     |
+| `auth-mode`      | `string`  | `prefix`   | The mode for authentication with the given service. Can be `prefix` or `structured_data`. Defaults to `prefix`.                                       |
 | `facility`       | `string`  | `1` (user) | A [syslog facility code](https://en.wikipedia.org/wiki/Syslog#Facility) to attach with each log to identify the source. Can be a number from 0 to 23. |
-| `message_format` | `string`  | `rfc5424`  | The standard to use for the message format. Can be `rfc5424` or `rfc3164`. |
+| `message-format` | `string`  | `rfc5424`  | The standard to use for the message format. Can be `rfc5424` or `rfc3164`.                                                                            |
 | `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`. |
```
Comment (CollierCZ) [inline docs/src/increase-observability/logs/forward-logs.md:125]:
```suggestion
| `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`.                                              |
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 5]
```diff
@@ -110,20 +110,20 @@ You can forward your Platform.sh logs to any of those endpoints.
...
 | `facility`       | `string`  | `1` (user) | A [syslog facility code](https://en.wikipedia.org/wiki/Syslog#Facility) to attach with each log to identify the source. Can be a number from 0 to 23. |
-| `message_format` | `string`  | `rfc5424`  | The standard to use for the message format. Can be `rfc5424` or `rfc3164`. |
-| `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`. |
-| `tls_verify`     | `boolean` | `true`     | Whether to verify Transport Layer Security (TLS) certification when using the TLS protocol. |
+| `message-format` | `string`  | `rfc5424`  | The standard to use for the message format. Can be `rfc5424` or `rfc3164`.                                                                            |
+| `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`.                                              |
```
Comment (CollierCZ) [inline docs/src/increase-observability/logs/forward-logs.md:125]:
```suggestion
| `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`.                                               |
```

**Review by CollierCZ — APPROVED** [review]

**Pull request merged**